event-management-frontend/src/components/eventCreateOrEdit/ticketsSection/CategoryPage/CategoryPageSmart.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { AppState } from "../../../../store/store";
+import { connect } from "react-redux";
+import CategoryPageDumb from "./CategoryPageDumb";
+import { EventCrud } from "../../../../model/EventCrud";
+import { addEmptyCategoryCard, updateEvent, updateFormErrors } from "../../../../actions/HeaderEventCrudActions";
+import { EventFormErrors } from "../../../../model/EventFormErrors";
+
+interface Props {
+  newEvent: boolean;
+  event: EventCrud;
+  formErrors: EventFormErrors;
+  addEmptyCategoryCard: () => void;
+  updateEvent: (event: EventCrud) => void;
+  updateFormErrors: (errors: EventFormErrors) => void;
+}
+
+const CategoryPageSmart: React.FC<Props> = (props: Props) => {
+  const handleChange = (e: any) => {
+    e.preventDefault();
+    const { name, value } = e.target;
+    // update event
+    let newEvent = Object.assign({}, props.event);
+
+    switch (name) {
+      case "ticketsPerUser":
+        newEvent.ticketsPerUser = parseInt(value);
+        break;
+      default:
+        break;
+    }
+    props.updateEvent(newEvent);
+
+    let newFormErrors = Object.assign({}, props.formErrors);
+
+    switch (name) {
+      case "ticketsPerUser":
+        newFormErrors.ticketsPerUser = value <= 0 ? "Add more people!" : "";
```

Comment:
String-ul "Add more people!" nu trebuie lasat aici. Pune-l in fisierele translation.json.